noir-projects/aztec-nr/aztec/src/macros/functions/mod.nr

```diff
@@ -87,6 +87,14 @@ pub comptime fn private(f: FunctionDefinition) -> Quoted {
     let fn_stub = stub_fn(f);
     register_stub(f.module(), fn_stub);
 
+    // If a function is further modified as unconstrained, we throw an error
```

Comment:
not using a function to marry these as the error message is different and its simple enough